tests/upgrades/test_contentview.py

```diff
@@ -84,21 +82,24 @@ def test_cv_postupgrade_scenario(self, request, dependent_scenario_name):
           3. The new repository should be added/updated to the CV.
 
         """
-        pre_test_name = dependent_scenario_name
-        org = entities.Organization().search(query={'search': f'name="{pre_test_name}_org"'})[0]
+        pre_test_name = pre_upgrade_data.get('test_name')
```

Comment:
Non-blocking : Can you please explain how this `pre_upgrade_data` works and fetch the test name from the `test_cv_preupgrade_scenario`?